benchmarking/process_json_to_csv.jl

```diff
@@ -44,23 +44,22 @@ import CSV
 import DataFrames
 import Glob
 import JSON3
-import StructArrays
 import StructTypes
 
 import FirstOrderLp
 
 const DataFrame = DataFrames.DataFrame
 
-function copy_fields_to_dict(to::Dict, from, fields::Vector{Symbol})
-  for fieldname in fields
-    to[fieldname] = getfield(from, fieldname)
-  end
-  return
+# Errors if "from" doesn't have the given fields.
+function extract_fields(from, fields::Vector{Symbol})
+  return Pair[fieldname => getfield(from, fieldname) for fieldname in fields]
```

Comment:
Is Pair necessary?  AFAICT, it isn't.

Reply:
Julia might choose a more specific element type like `Pair{Symbol, Float64}` depending on the data. This could create issues down the line if you try to append a `Pair{Symbol, String}` to the vector returned by this function. I don't know if this is an actual issue given how we use extract_fields, but it's easier to be on the defensive side.